etc/ci/buildbot_steps.yml

```diff
@@ -35,7 +35,7 @@ linux-rel-wpt:
     - ./mach test-wpt --release --binary-arg=--multiprocess --processes 24 --log-raw test-wpt-mp.log --log-errorsummary wpt-mp-errorsummary.log eventsource
     - ./mach test-wpt --release --product=servodriver --headless tests/wpt/mozilla/tests/mozilla/DOMParser.html tests/wpt/mozilla/tests/css/per_glyph_font_fallback_a.html tests/wpt/mozilla/tests/css/img_simple.html tests/wpt/mozilla/tests/mozilla/secure.https.html
     - ./mach test-wpt --release --product=servodriver --headless --log-raw test-bluetooth.log --log-errorsummary bluetooth-errorsummary.log bluetooth
-    - ./mach test-wpt --release --headless --timeout-multiplier=4 --log-raw test-wdspec.log --log-errorsummary wdspec-errorsummary.log --always-succeed webdriver
+    - HOST_FILE=tests/wpt/hosts ./mach test-wpt --release --headless --timeout-multiplier=4 --log-raw test-wdspec.log --log-errorsummary wdspec-errorsummary.log --always-succeed webdriver
```

Comment:
Rather than only setting this in CI, we should set this in https://github.com/servo/servo/blob/5788e8c050471121c927b597dfa7a06a065e8167/python/servo/testing_commands.py#L399 instead.